scripts/validate-notebooks.ps1

```diff
@@ -83,16 +83,24 @@ function Validate {
             </configuration>
         " | Out-File ./NuGet.Config -Encoding utf8
 
+		# Specifying tasks that need to be excluded because of various reasons :
+		# two_player_game : These tasks require implementing two code cells at once before running the test, so the first of the cells implemented is guaranteed to fail.
+		# failure_of_correct_solution : Tasks for which the correct solution fails with relatively high probability.
+		# timeout : Tasks for which the correct solution times out with relatively high probability.
+		# invalid_code : Tasks with deliberately invalid code
+		# work_in_progress : If the solutions to the tasks(Either in ReferenceImplementation or Workbook) are work_in_progress
+		$exclude_from_validation = "['two_player_game', 'failure_of_correct_solution', 'timeout', 'invalid_code', 'work_in_progress']"
```

Comment:
Please use spaces instead of tabs everywhere in this file

Reply:
Done.